server/monitor.go

```diff
@@ -53,6 +55,8 @@ type Serverz struct {
 	Channels      int       `json:"channels"`
 	TotalMsgs     int       `json:"total_msgs"`
 	TotalBytes    uint64    `json:"total_bytes"`
+	OpenFDs       int       `json:"open_fds"`
```

Comment:
I would change to `json:"open_fds,omitempty"` and same for max_fds. This would prevent the display of those values on systems other than linux. Or would that break your prometheus monitoring tool?

Reply:
not a problem at all :)